appeals/web/src/server/appeals/appeal-details/issue-decision/issue-decision.controller.js

```diff
@@ -78,9 +80,39 @@ export const renderIssueDecision = async (request, response) => {
 	});
 };
 
+/**
+ *
+ * @param {import("express-session").Session & Partial<import("express-session").SessionData>} session
+ * @param {string} documentType
+ */
+function storeFileUploadInfo(session, documentType) {
+	const { inspectorDecision = {} } = session;
+	if (!inspectorDecision.fileUploadInfo) {
+		inspectorDecision.fileUploadInfo = {};
+	}
+
+	if (session.fileUploadInfo) {
+		inspectorDecision.fileUploadInfo[documentType] = cloneDeep(session.fileUploadInfo);
+		session.inspectorDecision = inspectorDecision;
+	}
+}
+
+/**
+ *
+ * @param {import("express-session").Session & Partial<import("express-session").SessionData>} session
+ * @param {string} documentType
+ */
+function restoreFileUploadInfo(session, documentType) {
+	const { fileUploadInfo = {} } = session.inspectorDecision || {};
+
+	if (fileUploadInfo[documentType]) {
+		session.fileUploadInfo = fileUploadInfo[documentType];
```

Comment:
Similar question to the above, would it make sense to clear `session.inspectorDecision` once it's been moved?